torch/distributed/algorithms/ddp_comm_hooks/quantization_hooks.py

```diff
@@ -120,8 +121,9 @@ def quantization_perchannel_hook(
     process_group: dist.ProcessGroup, bucket: dist.GradBucket, bucket_size=512
 ) -> torch.futures.Future[torch.Tensor]:
     """
-    Applies the ``torch.quantize_per_channel`` logic to DDP using ``allgather``
-    protocol. Compared to pertensor, the main motivation of perchannel is
+    Apply``torch.quantize_per_channel`` logic to DDP using ``allgather`` protocol.
+
+    Compared to pertensor, the main motivation of perchannel is
```

Comment:
nit (before this PR):
```suggestion
    Compared to per-tensor, the main motivation of per-channel is
```